cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -243,11 +253,19 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   function getDom() {
     return {
       altTrigger: _altTriggerDom,
+      container:  _dom,
       content:    _contentDom,
       trigger:    _triggerDom
     };
   }
 
+  /**
+   * @returns {boolean} True if menu is expanded, false otherwise.
+   */
+  function isAnimating() {
```

Comment:
Added `TODO` in https://github.com/cfpb/cfgov-refresh/commit/3ee747cfba82bc15ac1040a50a4cab62363da076